src/hooks/useInfoTokens.ts

```diff
@@ -39,16 +39,39 @@ export function useInfoTokens(
     }
   );
 
-  const indexPricesUrl = getServerUrl(chainId, "/prices");
-  // const indexPricesUrl = `https://pricing.mycelium.xyz/prices?network=${chainId}`;
+  const oldIndexPricesUrl = getServerUrl(chainId, "/prices");
+  const newIndexPricesUrl = `https://pricing.mycelium.xyz/prices?network=${chainId}`;
 
-  const { data: indexPrices } = useSWR([indexPricesUrl], {
+  const { data: newIndexPrices } = useSWR([newIndexPricesUrl], {
     // @ts-ignore spread args incorrect type
     fetcher: (...args) => fetch(...args).then((res) => res.json()),
     refreshInterval: 500,
     refreshWhenHidden: true,
   });
 
+  const { data: oldIndexPrices } = useSWR([oldIndexPricesUrl], {
+    // @ts-ignore spread args incorrect type
+    fetcher: (...args) => fetch(...args).then((res) => res.json()),
+    refreshInterval: 500,
+    refreshWhenHidden: true,
+  });
+
+  let indexPrices: Record<string, BigNumber> = {};
+  if (newIndexPrices && oldIndexPrices) {
+    const FIVE_MINUTES = 60 * 5 * 1000;
+    let now = Date.now();
+    Object.keys(newIndexPrices).forEach((address) => {
+      const { medianAge, price } = newIndexPrices[address];
+      const isStale = Number(medianAge) < now - FIVE_MINUTES;
```

Comment:
waiting on confirmation of to approve this line

https://github.com/mycelium-ethereum/swaps-prices/pull/27/files/88dd591144a25f246f73dee5900b3beff2821f52..ec19bad9971e806355b8b06e17085b06fa9114f0#r1019836962

Reply:
confirmed